### PR TITLE
Add Event schema data for workshop 2024

### PR DESCRIPTION
**Pull request by fsimonis**
This PR adds [scheme.org Event data](https://schema.org/Event) for the Workshop 2024, which should make the Workshop discoverable by search engines as an Event.

Possible additions:
* `maximumAttendeeCapacity` 
* [`performers`](https://schema.org/performer): maybe interesting for the keynote speaker
* [`offers`](https://schema.org/performer): various tickets at different prices. Maybe too complicated.

### CODE REVIEW

**Review by carme-hp — COMMENTED** [review]
Thanks @fsimonis! It looks good, the only think that I can think to improve is a specific mention to the course: I saw you wrote "hands-on sessions", but since this year the course (24-25 September) and talks (25-27 September) are clearly separated (we event charge differently) it may be good to mention it specifically.  

I would be against adding the `maximumAttendeeCapacity` I think this is more of an internal decision to keep in mind.

File: pages/community/precice-workshop-2024.md [hunk 1]
```diff
@@ -12,3 +12,38 @@ redirect_from: /preCICE2024/
 The 5th preCICE Workshop will be held at the [University of Stuttgart](https://www.uni-stuttgart.de/), together with [SimTech](https://www.simtech.uni-stuttgart.de/), on September 24-27, 2024. The workshop is a coming together of the preCICE community to share ideas, experiences and knowledge about using preCICE, and to learn from others in the process. Like always, we plan to have user and developer talks, hands-on training sessions, discussions with the developers on your applications and use cases, and plenty of opportunities for networking.
 
 Keep watching this space for updates on registration, the workshop program, and more.
+
+<script type="application/ld+json">
+    {
+      "@context": "https://schema.org",
+      "@type": "Event",
+      "name": "preCICE Workshop 2024",
+      "description": "The preCICE workshop is a coming together of the preCICE community to share ideas, experiences and knowledge about using preCICE. Expect user and developer talks, hands-on training sessions, discussions with the developers on your applications and use cases, feedback rounds, and plenty of opportunities to network with the rest of our vibrant community.",
```
Comment (fsimonis) [inline pages/community/precice-workshop-2024.md]: @carme-hp Does this sound better?

```suggestion
      "description": "The preCICE workshop offers a platform for researchers and industry professionals to exchange insights, experiences, and expertise on leveraging preCICE. It includes an introductory course, user and developer presentations, interactive training sessions, and feedback rounds. Attendees can network with the dynamic community and consult developers on application strategies, use cases, and requirements.",
```

Comment (carme-hp) [reply pages/community/precice-workshop-2024.md]: Yes! It's very nice:)

File: pages/community/precice-workshop-2024.md [hunk 2]
```diff
@@ -12,3 +12,38 @@ redirect_from: /preCICE2024/
...
 
 Keep watching this space for updates on registration, the workshop program, and more.
+
+<script type="application/ld+json">
+    {
+      "@context": "https://schema.org",
+      "@type": "Event",
+      "name": "preCICE Workshop 2024",
+      "description": "The preCICE workshop offers a platform for researchers and industry professionals to exchange insights, experiences, and expertise on leveraging preCICE. It includes an introductory course, user and developer presentations, interactive training sessions, and feedback rounds. Attendees can network with the dynamic community and consult developers on application strategies, use cases, and requirements.",
```
Comment (IshaanDesai) [inline pages/community/precice-workshop-2024.md]:
```suggestion
      "description": "The preCICE workshop offers a platform for researchers and industry professionals to exchange insights, experiences, and expertise on leveraging preCICE. It includes an introductory course, user and developer presentations, interactive support sessions, and feedback rounds. Attendees can network with the dynamic community and consult developers on application strategies, use cases, and requirements.",
```

**Review by IshaanDesai — APPROVED** [review]
Looks good :+1: just a minor word change.

**Comment by fsimonis** [comment]
Great! Merging

**Pull request merged**